.gitignore

```diff
@@ -2,4 +2,5 @@ target/
 Cargo.lock
 .idea/
 *.iml
+*.tmp
```

Comment:
?

Reply:
I wanted to avoid checking in temporary files. I removed it.